### PR TITLE
fix: update README

### PR DESCRIPTION
**Pull request by andresmoschini**
Hi team!

I have updated the repo with a deprecation notice about _swarm-stack_ folder (I am not sure what to do to keep it updated) and with a link to our new private source of truth.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,11 @@
 
 Here we will store the configuration of our experimental Docker Swarm Architecture.
 
-And probably here will define the image for _Doppler Swarm Proxy_ with the required configuration.
+It also define the image for _Doppler Swarm Proxy_ with the required configuration.
+
+**IMPORTANT**: the folder _swarm-stack_ is very useful to test local, but its contents could be non-updated.
+
+See the production content in the repo [oppler-swarm-devops](https://github.com/MakingSense/doppler-swarm-devops).
```
Comment (elsupergomez) [inline README.md]:
```suggestion
See the production content in the repo [doppler-swarm-devops](https://github.com/MakingSense/doppler-swarm-devops).
```

File: README.md [hunk 2]
```diff
@@ -2,7 +2,11 @@
 
 Here we will store the configuration of our experimental Docker Swarm Architecture.
 
-And probably here will define the image for _Doppler Swarm Proxy_ with the required configuration.
+It also define the image for _Doppler Swarm Proxy_ with the required configuration.
+
+**IMPORTANT**: the folder _swarm-stack_ is very useful to test local, but its contents could be non-updated.
```
Comment (cbernat) [inline README.md]: So, all modifications we need in doppler stack now are to be deployed in the other repository, not just sensitive data right?

Comment (andresmoschini) [reply README.md]: Right, the scope of the [another repository (doppler-swarm-devops)](https://github.com/MakingSense/doppler-swarm-devops) is allowing us continuous deployment of our infrastructure (services, instances, certificates, production sensitive configuration data, etc).

The scope of the current repository is restricted to define our reverse proxy image.

The problem is how to test the complete environment for people without access to our _doppler-swarm-devops_ repository. Also, how to deal with QA environments.

Further work could be moving Nginx configuration to the _doppler-swarm-devops_ repository and dispense with this repository and the proxy image, but it does not solve the issue mentioned previously.

Comment (cbernat) [reply README.md]: Right so then I should update both for now. Mostly the other repository should be up with the latest changes. Thanks @andresmoschini

File: README.md [hunk 3]
```diff
@@ -2,7 +2,11 @@
 
 Here we will store the configuration of our experimental Docker Swarm Architecture.
 
-And probably here will define the image for _Doppler Swarm Proxy_ with the required configuration.
+It also define the image for _Doppler Swarm Proxy_ with the required configuration.
```
Comment (AlphaGit) [inline README.md]:
```suggestion
It also defines the image for _Doppler Swarm Proxy_ with the required configuration.
```

File: README.md (hunk 2 above)
Comment (AlphaGit) [inline README.md]:
```suggestion
**IMPORTANT**: the folder _swarm-stack_ is very useful to test locally, but its contents could be not up-to-date.
```

**Comment by andresmoschini** [comment]
Too many typos and gramatical issues in a very short change, sorry 😞

**Pull request merged**